subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -800,6 +800,9 @@ void ull_conn_setup(memq_link_t *link, struct node_rx_hdr *rx)
 
 	ftr = &(rx->rx_ftr);
 
+	/* NOTE: LLL conn context SHALL be after lll_hdr in
+	 *       struct lll_adv and struct lll_scan.
+	 */
 	lll = *((struct lll_conn **)((uint8_t *)ftr->param +
```

Comment:
Just a note for later discussion: Race with ADV disable may invalidate `ftr->param`, making it `NULL` by the time this is called. `lll->role` is then invalid.

Reply:
Agreed, do you have a fix already that you could send a PR?

Reply:
Create this to keep track: https://github.com/zephyrproject-rtos/zephyr/issues/33868